generate/generate.go

```diff
@@ -142,7 +142,14 @@ func New() Generator {
 					Type: "mount",
 				},
 			},
-			Devices: []rspec.Device{},
+			Devices: []rspec.Device{
+				{
+					Path:  "/dev/fuse",
```

Comment:
There's no need to add a value to the default config just so you can test device removal.  Test that by using a template config (although we don't have a test suite for the generator at the moment, see #180 for a stub).  But the criterion for adding an entry to `New()` should be “is this a useful part of a generic config?”, and I don't think `/dev/fuse` passes that bar.